components/playlist/mojom/playlist.mojom

```diff
@@ -39,24 +39,34 @@ struct PlaylistItem {
   int8 last_played_position;
 };
 
-// Used by the WebUI page to bootstrap bidirectional communication.
+// Used by the UI to bootstrap bidirectional communication.
 interface PageHandlerFactory {
-  // The WebUI page's |BrowserProxy| singleton calls this method when the page
-  // is first initialized.
-  CreatePageHandler(pending_remote<Page> page,
-                    pending_receiver<PageHandler> handler);
+  // On Desktop, the WebUI page's |BrowserProxy| singleton calls this method 
+  // when the page is first initialized.
+  BindInterface(pending_remote<ServiceObserver> service_observer,
```

Comment:
@sangwoo108 we can use the same function to bind android interfaces.

Reply:
Yes, I know. I guess you should call it manually.

Reply:
android doesn't use PageHandlerFactory, it is only for webui and this method should be named `CreatePageHandler` just like the others. Do not change the conventions.

Reply:
Oh, I think you agreed that what @simonhong suggested for this method name.

https://bravesoftware.slack.com/archives/C04G7RERW2J/p1671752113672259

Reply:
Done.